packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.tsx

```diff
@@ -23,13 +23,7 @@ function createSlotRegistry(): SlotFillBubblesVirtuallyContext {
 		ref,
 		fillProps
 	) => {
-		const slot = slots.get( name );
-
-		slots.set( name, {
-			...slot,
-			ref: ref || slot?.ref,
-			fillProps: fillProps || slot?.fillProps || {},
-		} );
+		slots.set( name, { ref, fillProps } );
```

Comment:
A drive-by change removing redundant code. All the `registerSlot` calls always include both the `ref` and `fillProps` values. Inheriting them from the previous `slot` value has no effect.